### PR TITLE
[Collocation] Ft device status table

### PR DESCRIPTION
**Pull request by Codebmk**
#### Summary of Changes (What does this PR do?)

- Features device status page
- Fixed top bar UI
- Reorganised redux folder structure

#### Status of maturity (all need to be checked before merging):

- [ ] I've tested this locally
- [ ] I consider this code done
- [ ] This change ready to hit production in its current state
- [ ] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [ ] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?

- Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [AN-303](https://airqoteam.atlassian.net/browse/AN-303)

#### Screenshots (optional)
image

[AN-303]: https://airqoteam.atlassian.net/browse/AN-303?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Baalmart — COMMENTED** [review]
Thanks @Codebmk 

**Context**: See attached screenshot.

Do we have any naming conventions we are following?  If yes, please shed some light on it.

Is Camelcase or underscore?

image

**Comment by Codebmk** [comment]
@Baalmart yes we do. Convention depends on the role of that file eg. Files under the pages folder use underscores and not camel case because next js uses the file name as the routes name. Components use both camel case.

**Comment by Baalmart** [comment]
Quick one @Codebmk , have we yet designed for responsiveness?

See attached screenshot...

image

**Comment by Codebmk** [comment]
@Baalmart I have fixed the collocate header responsiveness. Thanks.

**Comment by Codebmk** [comment]
@Baalmart I have some new changes to add responsiveness but Paul hasn't designed it for mobile yet.

**Comment by Baalmart** [comment]
> @Baalmart I have some new changes to add responsiveness but Paul hasn't designed it for mobile yet. Changes can be seen in deploy preview.

Thanks @Codebmk !

**Review by Baalmart — APPROVED** [review]
Thaks @Codebmk , all looks good to me.

File: platform/src/pages/analytics/airqlouds.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import Layout from '@/components/Layout';
 
 const AirQlouds = () => (
   <Layout>
-    <div>I am the airqlouds content</div>
+    <div></div>
```
Comment (123MwanjeMike) [inline platform/src/pages/analytics/airqlouds.jsx:5]: Hi @Codebmk 
Does this empty div have a purposes?

File: platform/src/pages/index.jsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import Layout from '@/components/Layout';
 
 const HomePage = () => (
   <Layout>
-    <div>I am the home page</div>
+    <div></div>
```
Comment (123MwanjeMike) [inline platform/src/pages/index.jsx:5]: Same as this one?

**Review by 123MwanjeMike — APPROVED** [review]
Thank you @Codebmk 
The changes look good to me
I have however seen a few empty divs, perhaps you could shed some light on them if possible

File: platform/src/pages/index.jsx (hunk 2 above)
Comment (Codebmk) [reply platform/src/pages/index.jsx:5]: Yeah. That's dummy code. Doesn't affect anything. Removed the text(highlighted in red) as there was a suggestion to leave those pages empty.

**Review by FaithDaka — APPROVED** [review]

**Pull request merged**